drivers/pci/quirks.c

```diff
@@ -267,28 +267,27 @@ static int pcie_acs_overrides(struct pci_dev *dev, u16 acs_flags)
 {
 	int i;
 
-	/* Never override ACS for legacy devices or devices with ACS caps */
-	if (!pci_is_pcie(dev) ||
-		pci_find_ext_capability(dev, PCI_EXT_CAP_ID_ACS))
+	/* Never override ACS for legacy devices */
+	if (!pci_is_pcie(dev))
 			return -ENOTTY;
 
 	for (i = 0; i < max_acs_id; i++)
 		if (acs_on_ids[i].vendor == dev->vendor &&
 			acs_on_ids[i].device == dev->device)
 				return 1;
 
-switch (pci_pcie_type(dev)) {
-	case PCI_EXP_TYPE_DOWNSTREAM:
-	case PCI_EXP_TYPE_ROOT_PORT:
-		if (acs_on_downstream)
-			return 1;
-		break;
-	case PCI_EXP_TYPE_ENDPOINT:
-	case PCI_EXP_TYPE_UPSTREAM:
-	case PCI_EXP_TYPE_LEG_END:
-	case PCI_EXP_TYPE_RC_END:
-		if (acs_on_multifunction && dev->multifunction)
-			return 1;
+	switch (pci_pcie_type(dev)) {
```

Comment:
@rucoder , so here are you changing only the indentation, right? But in the kernel coding standard both switch and case statements must be aligned, so in this case you just need to fix the `switch()` line...

Reply:
too late :) I'm just merging a PR here